db/migrate/20161116114622_remove_invalid_draft_content_item.rb

```diff
@@ -0,0 +1,10 @@
+require_relative "helpers/delete_content_item"
+
+class RemoveInvalidDraftContentItem < ActiveRecord::Migration[5.0]
+  def change
```

Comment:
Yes, I think it could be a good idea to split this into `up` and `down`. But I am wondering if this should raise an IrreversibleMigration error on down or simply do nothing... If it manages also the situation where the content item doesn't exist, maybe doing nothing is acceptable too and it doesn't creates problems in case someone needs to rollback to a previous migration. What do you think?